docs/source/drush_commands.rst

```diff
@@ -123,6 +123,11 @@ dkan:harvest:deregister
     **Arguments**
```

Comment:
Let's define this a little more on line 121.

Deregister a harvest. This action removes the harvest plan from the system, but will leave the datasets in place. This is good if you are using harvest to migrate content and you do not plan on using the harvest feature to update the datasets. 
If you DO want to remove the harvest plan AND the datasets associated with it, uses the `--revert` flag.

Reply:
Also update the dkan:harvest:revert description

Revert a harvest, i.e. remove harvested entities and unpublish orhpaned keywords, themes, and distributions. The harvest plan will remain and can be run again to generate the datasets after any issues have been resolved.